kotlinx-coroutines-core/jvm/src/Interruptible.kt

```diff
@@ -4,77 +4,65 @@
 
 package kotlinx.coroutines
 
-import kotlinx.atomicfu.AtomicRef
-import kotlinx.atomicfu.atomic
-import kotlinx.atomicfu.loop
-import kotlin.coroutines.CoroutineContext
-import kotlin.coroutines.EmptyCoroutineContext
-import kotlin.coroutines.intrinsics.suspendCoroutineUninterceptedOrReturn
+import kotlinx.atomicfu.*
+import kotlin.coroutines.*
 
 /**
- * Makes a blocking code block cancellable (become a cancellation point of the coroutine).
- *
+ * Calls the specified [block] with a given coroutine context in a interruptible manner.
  * The blocking code block will be interrupted and this function will throw [CancellationException]
  * if the coroutine is cancelled.
+ * The specified [coroutineContext] directly translates into [withContext] argument.
  *
  * Example:
  * ```
- * GlobalScope.launch(Dispatchers.IO) {
- *     async {
- *         // This function will throw [CancellationException].
- *         runInterruptible {
- *             doSomethingUseful()
- *
- *             // This blocking procedure will be interrupted when this coroutine is canceled
- *             // by Exception thrown by the below async block.
- *             doSomethingElseUsefulInterruptible()
- *         }
+ * val blockingJob = launch {
+ *     // This function will throw [CancellationException].
+ *     runInterruptible(Dispatchers.IO) {
+ *         // This blocking procedure will be interrupted when this coroutine is canceled
+ *         // by Exception thrown by the below async block.
```

Comment:
// by Exception thrown by the below async block. 
Forget to change this comment line as well?